bp_common/src/include/bp_common_aviary_defines.vh

```diff
@@ -229,9 +231,11 @@ typedef struct packed
   , localparam dcache_sets_p              = proc_param_lp.dcache_sets                              \
   , localparam dcache_assoc_p             = proc_param_lp.dcache_assoc                             \
   , localparam dcache_block_width_p       = proc_param_lp.dcache_block_width                       \
+  , localparam dcache_fill_width_p        = proc_param_lp.dcache_fill_width                        \
   , localparam icache_sets_p              = proc_param_lp.icache_sets                              \
   , localparam icache_assoc_p             = proc_param_lp.icache_assoc                             \
   , localparam icache_block_width_p       = proc_param_lp.icache_block_width                       \
+  , localparam icache_fill_width_p        = proc_param_lp.icache_fill_width                        \
```

Comment:
Good, also add an assertion to bp_nonsynth_if_verif that fill_width == block_width